plugins/tekton/src/utils/tekton-utils.ts

```diff
@@ -17,6 +17,11 @@ import {
   pipelineRunStatus,
 } from './pipeline-filter-reducer';
 
+export type TektonResources = {
```

Comment:
nit: if it's not used out of this file then don't need to export it or can move it to types